example/glue/EnvironmentLanguageInterfaceImpl.cpp

```diff
@@ -74,12 +74,17 @@ MM_EnvironmentLanguageInterfaceImpl::tearDown(MM_EnvironmentBase *env)
 bool
 MM_EnvironmentLanguageInterfaceImpl::saveObjects(omrobjectptr_t objectPtr)
 {
+	Assert_MM_true(NULL == _omrThread->_savedObject);
```

Comment:
This should be on the Environment and not the EnvironmentLanguageInterface so that all languages get the support